event-listeners-mandatory-arguments/index.html

```diff
@@ -0,0 +1,18 @@
+---
+feature_name: addEventListener/removeEventListener non-optional arguments
+chrome_version: 49
+feature_id: 5640816202612736
+---
+
+<h3>Background</h3>
+<p> For a very long time, the first two arguments (<code>type</code> and
+<code>listener</code>) of <code>addEventListener</code> and
```

Comment:
Link to https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener